docs/trusted-signing-integration.md

```diff
@@ -0,0 +1,98 @@
+# Trusted Signing integration for Sign CLI
+
+This document explains how to use the Sign CLI with a Trusted Signing account to perform code signing using the Trusted Signing provider. See `docs/signing-tool-spec.md` for higher-level background of this tool and the implementation at `src/Sign.SignatureProviders.TrustedSigning` for details.
+
+## Overview
+
+The Sign CLI includes a `trusted-signing` provider that invokes the Trusted Signing service to obtain certificates and perform remote sign operations. The CLI uses the Azure SDK (`Azure.Identity`) for authentication.
+
+Key concepts for this provider:
+- Endpoint: the service URL for the Trusted Signing account.
+- Account name: the account within the Trusted Signing service.
+- Certificate profile: the certificate profile configured in the account that will be used to sign.
+
+For more information, see the Trusted Signing [setup documentation](https://learn.microsoft.com/en-us/azure/trusted-signing/quickstart).
```

Comment:
```suggestion
For more information, see the Trusted Signing [setup documentation](https://learn.microsoft.com/azure/trusted-signing/quickstart).
```

This allows the site to redirect the user to their preferred language based on browser settings.